dist/mapdc.js

```diff
@@ -60394,13 +60394,9 @@ function filterHandlerWithChartContext(_chart) {
       if (_chart.clearTableFilter) {
         _chart.clearTableFilter(); // global filter also will clear all the columns filters on the table
       }
-    } else if (_chart.hasOwnProperty("rangeFocused")) {
-      dimension.filterMulti(filters, _chart.rangeFocused(), _chart.filtersInverse(), _chart.group().binParams());
     } else if (_chart.getFilteredColumns && Object.keys(_chart.getFilteredColumns()).length > 0) {
```

Comment:
`filterMulti` is now a no op, so just nuke these calls to it. These were the source of the _chart.group().binParams() does not exist errors.